### ISSUE TITLE
only black screen after following instructions for 4K - solution(?)

### ISSUE THREAD

**Issue by sn00zerman** [Floris497/mac-pixel-clock-patch-V2#157]
Hi all,

I just want to let you know, that I struggled with a black screen after following the instructions on both a Macbook Pro Retina early 2013 and a Mac Mini late 2012. (it was working fine before the Sierra update) 
The solution seems easy :-) I selected 1080P24Hz in SwitchResX, this does give a "workable" resolution. (but not 4K yet !) Make sure that the wanted resolution shows up in SwitchResX.
(I got this by using the IOKit patch v3 command) 
Next, I disabled the SwitchResX daemon (and also disabled it for startup), 
and opened up the default MacOS screen settings.
You will also see the 4K resolution show up in that screen. Select you refresh rate, and next select your wanted resolution. You should now be able to use the 4K ...

(I used Sierra 10.12.3 and SwitchResX 4.6.3 with the latest IOKit Patch Command)

I hope anyone can confirm this as a working solution on his setup ?

best regards,
Kris
http://www.digitalplayground.be
Where fun meets technology ...

**Comment by Floris497** [comment]
Why not use CoreDisplay patch?

**Comment by sn00zerman** [comment]
Floris, you are right, I'm using CoreDisplay. (I still have problems with my brain after beeing hospitalized in the beginning of this year, I even did spend 3 weeks in coma !)
I did notice a few messages from people with the same problem (only black screen after selecting an alternative resolution), maybe this could help them ?

best regards,
Kris
http://www.digitalplayground.be
Where fun meets technology ...

**Comment by Floris497** [comment]
Thanks I'm loosing a bit of track so many little issues with different versions of macOS and different hardware.. I will have to see to make a troubleshooting guide for most common problems. 
I do think your mac actually both can do 4k@30Hz at least.. i would recommend using a displayport cable if your screen supports it. if you want to get 4k@60Hz over HDMI make sure you have an DP to HDMI2.0 adapter.. there are only a few.. the package saying 4k ready or 4k@60Hz is mostly not a guarantee it's HDMI2.0

I hope you are doing better now :)

**Comment by AlexiDime** [comment]
yeah I do have the same issue and I've already opened a ticket were Floris helped me but I couldn't solve the problem yet. I'm at the point were I consider selling my macbook and building a hackintosh or buying a 2016 mbp even if I totally hate everything on it but 24Hz on my 55" 4k Screen is freaking me out. I would pay a person fixing this for me just to spare me buying a new mac or building a hack.
